llama-index-integrations/tools/llama-index-tools-yahoo-finance/llama_index/tools/yahoo_finance/base.py

```diff
@@ -0,0 +1,92 @@
+from llama_index.core.tools.tool_spec.base import BaseToolSpec
+import yfinance as yf
```

Comment:
can we add this dep to the pyproject.toml? (and can you run `poetry lock` after that and commit the lockfile?

Reply:
Yes just did that.